### ISSUE TITLE
Get LoraLoader lora name as text

### ISSUE THREAD

**Issue by Eliont** [comfyanonymous/ComfyUI#561]
Hello and good time of day.

How to get element value as text to set as SaveImage filename prefix ?

**Comment by WASasquatch** [comment]
But the LoraLoader isn't dynamic, you set it manually, so why would it be so hard to just set a filename prefix when changing your Lora? 

And if you mean like to save the trigger word to the filename, Loras often have many trigger tokens.

**Comment by Eliont** [comment]
Yes, need to auto set SaveImage header to text like AUTO1111 can name output files.

Well, at least need lora name.

**Comment by WASasquatch** [comment]
See commit: https://github.com/WASasquatch/was-node-suite-comfyui/commit/645311955255c7de8ac5a6ce01d240306a709e3a

**Comment by Eliont** [comment]
> I can add these features to custom loaders for WAS Node Suite if you'd like. Not sure if Comfy would want to add this as it seems like a very special case use.

Thanks, lora loader with NAME_STRING output exacly what is needed atm.

**Comment by SebAnt333** [comment]
Thanks, lora loader with NAME_STRING output what I needed. I added 4 lora loaders in sequence, and sent the 4 text outputs to the text concatenate node, using | as delimiter.  Then I grouped all 4 loaders and text-concatenate into a group node.  Found that selecting lora type none throws an error on the group node - 'NoneType' object has no attribute 'lower'. However, I was able to resolve by choosing random lora and setting the strength to exactly 0.  So this is a bit like the rgthree power lora loader, but more useful for me due to the ability to see chosen Lora names that I can optionally overlay into final image.
